modules/disk-virt-customize/build/disk-virt-customize/Dockerfile

```diff
@@ -1,11 +1,9 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal AS taskBuilder
-RUN microdnf install -y tar gzip && microdnf clean all
+RUN microdnf install -y tar gzip golang && microdnf clean all
```

Comment:
Similar to above, how do we make sure the golang version matches the enabled golang feature set in go.mod?